src/mixs/schema/mixs.yaml

```diff
@@ -1179,6 +1179,49 @@ enums:
       middle:
       top:
 slots:
+  marker_gene_recov:
+    description: Recovery status of taxonomic marker genes from the assembled genome
+    title: taxonomic marker gene recovered
+    examples:
+      - value: 'yes'
+    in_subset:
+      - sequencing
+    keywords:
+      - recover
+      - marker gene
+      - taxonomy
+    slot_uri: MIXS:0001337
+    range: boolean
```

Comment:
maybe I'm just crazy.. I thought boolean was TRUE FALSE but you have yes.. so wouldn't this need to be a yes / no enum?